tomls/omnibus-arbitrum-mainnet/spot/btc.toml

```diff
@@ -119,4 +119,4 @@ target = ["spotFactory.SpotMarketProxy"]
 fromCall.func = "getMarketOwner"
 fromCall.args = ["<%= extras.synth_btc_market_id %>"]
 func = "setAtomicFixedFee"
-args = ["<%= extras.synth_btc_market_id %>", "<%= parseEther(String(0.01)) %>"]
```

Comment:
Regardless of how well we document this, a user will end up touching it and losing funds due to the large fee. We really need a better solution for this.

Reply:
i'd direct these concerns to @cavalier-eth  who set the priority low

Reply:
Will do, thank you.